src/plugin/protocol-foo/src/java/my/foo/Foo.java

```diff
@@ -0,0 +1,126 @@
+package my.foo;

```

Comment:
Would it be possible to change package name to org.apache.nutch.foo? We will be publishing these plugins to Maven Central at some stage in the future and a consistent namespace would be preferred.